lib/locales/es.ts

```diff
@@ -171,8 +171,8 @@ export const locale: Locale = {
     adalbert_of_prague_bishop: 'San Adalberto, obispo y mártir',
     adelaide_of_burgundy_empress: 'Santa Adelaida de Borgoña, Emperatriz',
     adelphus_of_metz_bishop: 'San Adelfus de Métis, obispo',
-    agatha_of_sicily_virgin: 'Santa Ágatha, virgen y mártir',
-    agnes_of_rome_virgin: 'Santa Agnes, virgen y mártir',
+    agatha_of_sicily_virgin: 'Santa Ágata, virgen y mártir', // Vatican news source: https://www.vaticannews.va/es/santos/02/05/s--agata--virgen-y-martir-de-catania.html
```

Comment:
According to [this](https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=4), it should be `Águeda` instead of `Ágata`.

Anyway, I suggest to format the source comment as `src: $source_title, $chapter_or_section_title, $page_number` (`$page_number` is for printed media and `$source_title` for a Roman Missal translation should be `mr_${locale}_${publish_year}_ed${edition_number}_${differentiator}` where `${differentiator}` is optional and should be used when multiple Roman Missal versions co-exist in a particular locale, e.g. for `en`, `es`; e.g. `mr_en_2011_ed3_us` for the English translation of the Roman Missal, ed. typ. tertia, used in the US and published in 2011). Moreover, when a particular celebration name is not found in full in the source, `src` should be replaced with `based on`. Thanks for understanding! :pray:

Reply:
Updated, done!